database/connector.py

```diff
@@ -178,6 +179,33 @@ def insert_honeypot_events_credential_from_module_processor(ip, username, passwo
     ).inserted_id
 
 
+def insert_file_change_events(file_path, status, module_name, date):
```

Comment:
There are 2 functions here with exactly the same name and same function. Is this a merge error?